configuration/pih/htmlforms/hiv/section-hiv-plan.xml

```diff
@@ -1168,47 +1180,48 @@
         </div>
     </section>
 
-    <section id="transfer-out-section" sectionTag="fieldset"
-             headerTag="legend" headerStyle="title" headerCode="pihcore.transfer.title">
+    <section id="transferIn" sectionTag="fieldset" headerTag="legend" headerStyle="title"
+             headerCode="pihcore.transferReferralIn">
         <div class="section-container-color">
-            <obsgroup groupingConceptId="PIH:13170">
```

Comment:
This is the question about transferring/referring *out* on the plan sectoin of the HIV form.  I didn't realize we were doing transfer out as well as in.  

It's fine to do that, but we we shouldn't change the grouping concept here... I didn't realize that we wanted to use this new question for both transfers out and transfers in... I will update "PIH:13170" concept set to have the "referral or transfer" concept as well.